### ISSUE TITLE
Calculation error when check in 

### ISSUE THREAD

**Issue by wolfwolfgod** [orvice/ss-panel#399]
when current transfer_enable reaches or exceeds  the maximum of Integer, then do check in will case calculation error. I think it because the casts in Eloquent Modal change this column to int type in PHP. 
We should directly use attributes do calculation.

**Comment by orvice** [comment]
in 64bits os:
```
<?php
$large_number = 9223372036854775807;
var_dump($large_number);                     // int(9223372036854775807)
```
I think 8589934592GB is enough.

**Comment by wolfwolfgod** [comment]
but in 32bits os, 

$large_number = 2147483647;
var_dump($large_number);                     // int(2147483647)

less then 2G

I know this issue because it happen in my 32bits VPS. 
I have found these is a  "addTraffic" method which is not implemented  in User.php. I guess that's the right place to do it. :)

**Comment by orvice** [comment]
fixed in v3.3.7
https://github.com/orvice/ss-panel/releases/tag/v3.3.7
